pages/linux/acpi.md

```diff
@@ -0,0 +1,23 @@
+# acpi
+
+> Shows battery status or thermal information.
+
+- Show battery information:
+
+`acpi`
+
+- Show thermal information:
+
+`acpi -t`
+
+- Show cooling device information:
+
+`acpi -c`
+
+- Show thermal information using Fahrenheit:
```

Comment:
Perhaps this will read better?
```suggestion
- Show thermal information in fahrenheit:
```

Reply:
Shouldn't the F be in capital case ?

Reply:
Good catch, @agnivade - I've checked and that is indeed the case. I've updated my suggestion.

Reply:
Made the change myself :)